raitools/raitools/_managers/explainer_manager.py

```diff
@@ -117,7 +132,24 @@ def get(self):
             return []
 
     def list(self):
-        pass
+        """List information about the ExplainerManager.
+
+        :return: A dictionary of properties.
+        :rtype: dict
+        """
+        props = {ListProperties.MANAGER_TYPE: self.name}
+        if self._explanation:
+            props[Keys.ID] = self._explanation.id
+            props[Keys.METHOD] = self._explanation.method
+            props[Keys.MODEL_TASK] = self._explanation.model_task
+            props[Keys.MODEL_TYPE] = self._explanation.model_type
+            if FeatureImportanceExplanation._does_quack(self._explanation):
+                props[Keys.IS_RAW] = self._explanation.is_raw
```

Comment:
Is one of raw or engineered enough? Why add two?

Reply:
explanations can be raw, engineered, or neither?

Reply:
I thought that explanations could be either raw/engineered. Is that not so?

Reply:
it can be neither too, the flag will not be set if transformations/feature maps was not passed in.  We have a bunch of tests that validate this too, for example see:

```
    def test_get_global_raw_explanations_classification(self, iris, tabular_explainer):
        model = create_sklearn_svm_classifier(iris[DatasetConstants.X_TRAIN], iris[DatasetConstants.Y_TRAIN])

        exp = tabular_explainer(model, iris[DatasetConstants.X_TRAIN], features=iris[DatasetConstants.FEATURES],
                                classes=iris[DatasetConstants.CLASSES])

        global_explanation = exp.explain_global(iris[DatasetConstants.X_TEST])
        assert not global_explanation.is_raw
        assert not global_explanation.is_engineered
```

in interpret-community\test\raw_explain\test_raw_explanation.py:

https://github.com/interpretml/interpret-community/blob/master/test/raw_explain/test_raw_explanation.py#L28

in fact there are even more weird cases where it can be neither, for example if it's not a FeatureImportanceExplanation:

https://github.com/interpretml/interpret-community/blob/master/python/interpret_community/explanation/explanation.py#L1805

although technically all of them are currently

Reply:
Thanks. Didn't know that